types/2020-03-02/SourceMandateNotifications.d.ts

```diff
@@ -14,6 +14,8 @@ declare module 'stripe' {
        */
       object: 'source_mandate_notification';
 
+      acss_debit?: SourceMandateNotification.AcssDebit;
```

Comment:
This is in the spec for legacy reason even though it shouldn't be and fixing it is "too hard" so let's just ignore this part